### PR TITLE
Add copyright

### PR DESCRIPTION
**Pull request by CuriousVini**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,17 @@
+Copyright © 2017-2018 Cask Data, Inc.
```
Comment (albertshau) [inline LICENSE]: we don't need copyright in the license do we?

**Review by albertshau — COMMENTED** [review]
can you also modify the project so that it has a rat check?

File: LICENSE (hunk 1 above)
Comment (CuriousVini) [reply LICENSE]: In cdap repo we have it: https://github.com/caskdata/cdap/blob/develop/LICENSE.txt#L1

Comment (albertshau) [reply LICENSE]: that seems odd... are we sure that's correct in the CDAP repo?

**Review by albertshau — APPROVED** [review]
lgtm

**Comment by CuriousVini** [comment]
squashing and merging.

**Pull request merged**